src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json

```diff
@@ -148,10 +148,11 @@
   },
   {
     "id": "Microsoft.Azure.WebJobs.Extensions.Sql",
-    "majorVersion": "0",
+    "majorVersion": "1",
     "name": "SqlBinding",
     "bindings": [
-      "Sql"
+      "Sql",
+      "SqlTrigger"

```

Comment:
@soninaren - Where is this value from?

Reply:
@Charles-Gagnon this would need to match the name of the bindings/triggers provided by the extension. This would be the same names you'd expect users to provide in `function.json` when using the bindings.

Reply:
Thanks, just verified and SqlTrigger is correct (I assume it's not case sensitive since `sql` works just fine in the function.json)